merge.go

```diff
@@ -30,7 +30,7 @@ type Config struct {
 	AppendSlice                  bool
 	TypeCheck                    bool
 	Transformers                 Transformers
-	overwriteWithEmptyValue      bool
+	OverwriteWithEmptyValue      bool
```

Comment:
As commented in the previous PR, this doesn't need to be exported (*capitalize*) :angel: `WithOverwriteWith

Reply:
okay. got it